chapters/playingenvironment.adoc

```diff
@@ -34,20 +34,24 @@ Distances between lines are measured from their centers.
 ===== Field Lines
 The playing area is defined by four field lines. The two longer field lines are called touch lines. The two shorter field lines are called goal lines.
 
-===== Additional Lines
-The field of play is divided into two halves by a halfway line that runs along the width of the field and through the center of the field. The halfway line is parallel to the goal lines.
+===== Halfway Line
+The field of play is divided into two halves by a  halfway line that runs along the width of the field and through the center of the field.
```

Comment:
There are two spaces between "a" and "halfway".  Please remove one of them.

Reply:
Sharp eyes! Thx :+1: 
```suggestion
The field of play is divided into two halves by a halfway line that runs along the width of the field and through the center of the field.
```